includes/class-newspack-popups-model.php

```diff
@@ -414,13 +414,13 @@ public static function has_newsletter_prompt( $popup ) {
 	 * @return string Prints the generated amp-analytics element.
 	 */
 	protected static function insert_event_tracking( $popup, $body, $element_id ) {
```

Comment:
PHPCS nit: missing docblock `@param` for`$body`.

Reply:
Done in 856d73618a30aca8f7940edb5b4983225d675b97